EIPS/eip-4899.md

```diff
@@ -0,0 +1,93 @@
+---
+eip: 4899
```

Comment:
```suggestion
eip: 4955
```
Please also update the filename and the PR title and the discussions-to link with the updated EIP number.